src/pages/atomicdex/api/v20-dev/non_fungible_tokens/index.mdx

```diff
@@ -0,0 +1,762 @@
+# Non Fungible Tokens (NFTs)
+
+The AtomicDEX API supports [ERC1155](https://www.nftstandards.wtf/Standards/ERC1155+Multi+token) and [ERC721](https://www.nftstandards.wtf/Standards/ERC721+Non+Fungible+Standard) NFTs via the [Moralis API](https://docs.moralis.io/) on the Avalanche (AVAX), BNB Smart Chain (BNB), Ethereum (ETH), Fantom (FTM), Polygon (MATIC) networks.
+
+
+
+## NFT Information Methods
+
+- Get a list of your tokens with [get_nft_list](/atomicdex/api/v20-dev/non-fungible-tokens/get-nft-list)
+- Get a list of token transfers with [get_nft_transfers](/atomicdex/api/v20-dev/non-fungible-tokens/get-nft-transfers)
+- Get token metadata with [get_nft_metadata](/atomicdex/api/v20-dev/non-fungible-tokens/get-nft-metadata)
+
+
+## NFT Transaction Methods
+
+- Withdraw ERC721 tokens with [withdraw_nft](/atomicdex/api/v20-dev/non-fungible-tokens/withdraw-nft#erc721)
+- Withdraw ERC1155 tokens with [withdraw_nft](/atomicdex/api/v20-dev/non-fungible-tokens/withdraw-nft#erc1155)
+
+
+## Get a list of NFTs {{label : 'get_nft_list', tag : 'API-v2'}}
+
+Returns a list of the NFTs owned by the user, shown in descending order of the `block_number` value (the block height when the amount or owner changed). If the request is for NFTs on more than one chain, this means that the order may not be chronological.
+
+### Request Parameters
+
+| Parameter     | Type             | Description                                                                                               |
+| ------------- | ---------------- | ----------------------------------------------------------------------------------------------------------|
+| chains        | array            | List of chains to scan for NFTs.                                                                          |
+| max           | boolean          | Optional, defaults to `false`. If `true`, response will return all NFTs without pagination, and will ignore the `limit` and `page_number` values.               |
+| limit         | integer          | Optional, defaults to `10`. The number of NFTs displayed per page in response.                            |
+| page_number   | integer          | Optional, defaults to `0`. The page offset for items in response.                                         |
```

Comment:
we use nonzero usize type for page_number, you will have this err, if you set 0 page

```
{"mmrpc":"2.0","error":"Error parsing request: invalid value: integer `0`, expected a nonzero usize","error_path":"dispatcher","error_trace":"dispatcher:109]","error_type":"InvalidRequest","error_data":"invalid value: integer `0`, expected a nonzero usize","id":null}
```

according to this implementation, it doesnt matter provided user 1 page or not. So we can say that 1st page is default.
```rust
let (offset, limit) = if max {
                (0, count_total)
            } else {
                match page_number {
                    Some(page) => ((page.get() - 1) * limit, limit),
                    None => (0, limit),
                }
            };
```